docker-compose.yml

```diff
@@ -559,6 +559,11 @@ services:
     depends_on:
       - ubuntu
 
+  rabbitmq36_management:
+    build:
+      context: ./rabbitmq/
+    image: quay.io/continuouspipe/rabbitmq36-management:latest
```

Comment:
Is it possible to declare an external_rabbitmq36_management that this depends on too? Not sure we still use the build.sh script which pulls any external_* beforehand, but might be useful still.

Reply:
added external dependency